tests/cases/fourslash/formatTypeArgumentOnNewLine.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="fourslash.ts"/>
+
+////const genericObject = new GenericObject<
+////  /*1*/{}

```

Comment:
this is the important one, right? Has anything changed for 2 or 3?

Reply:
No change for 2 or 3 (as they were also blocked by the crash (since type argument is on different line from arguments)  I just added another test when there are multiple type arguments..